config/data/langs.toml

```diff
@@ -883,6 +883,29 @@ example = '''
 `0:x
 '''
 
+[Lil]
+args       = [ '/usr/bin/lilt', '-e', '$code' ]
+experiment = 1174
+size       = '1.67 MiB'
+version    = 'Lilt 1.48'
+website    = 'https://beyondloom.com/decker/lilt.html'
+example    = '''
+# Printing
+print["Hello, World!"]
+
+# Looping
+each i in range 10
+    print[i]
+end
+
+# Accessing arguments
+each i in range count args
+  if i > 2
+    print[args[i]]
+  end
+end
+'''
```

Comment:
In all honesty, I couldn't be arsed any longer for such a tiny pee pee language, or should I say lil? And why does all that small shit run fifty solutions while Kotlin figures out what to do...